packages/bpk-react-utils/src/changeFavicon.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// This util was inspired by this answer on SO:
+// https://stackoverflow.com/a/2995536
+
+/*!
+ * Dynamically changing favicons with JavaScript
+ * Works in all A-grade browsers except Safari and Internet Explorer
+ * Demo: http://mathiasbynens.be/demo/dynamic-favicons
+ */
+
+const changeFavicon = src => {
+  if (!document) {
+    return;
+  }
+
+  document.head = document.head || document.getElementsByTagName('head')[0];
+
+  const link = document.createElement('link');
+  const oldLink = document.getElementById('dynamic-favicon');
+
+  link.id = 'dynamic-favicon';
+  link.rel = 'shortcut icon';
```

Comment:
Would it maybe be good to allow this be configured incase it changes rather than hardcoding?